sourcegraph/sourcegraph
# [SG-27703] Wildcard: `<NavMenu />` Implementation

## Description
We already have `<Menu />`, `<MenuDivider />` accessible components here: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@tr/wildcard-popover/-/blob/client/wildcard/src/components/Menu/Menu.tsx#L1:1
We want to be able to easily implement navigation menus

## Test Plan
- Visit the app storybook preview link
- Confirm the the `NavMenu` story matches desired result.
- Confirm that Component has all required APIs that are needed by consumer.

## Ref 
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/27703)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-27703)

## Preview App
[Web App](https://client-sourcegraph-frontend-pr-286.onrender.com/)
[Storybook](https://client-sourcegraph-storybook-pr-286.onrender.com/)

Acceptance criteria
 - [x] Component is implemented within Wildcard
 - [x] Component has tests
 - [x] Component is accessible
 - [x] Component matches [Figma designs](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Wildcard-Design-System?node-id=908%3A1)

## Time Estimate
10hrs - 12hrs
## App preview:
- [Link](https://sg-web-contractors-sg-27703.onrender.com)
